package.json

```
@@ -5,10 +5,10 @@
   "description": "Dashboard for Optuna",
   "main": "index.js",
   "scripts": {
-    "fmt": "prettier --write \"{optuna_dashboard/ts,typescript_tests,standalone_app/src,vscode/src}/**/*.{ts,tsx}\"",
+    "fmt": "npx @biomejs/biome format --write .",
```

Probably, we could just call `biome` CLI entrypoint in npm run scripts.

```suggestion
    "fmt": "biome format --write .",
```